settings.sh

```diff
@@ -0,0 +1,5 @@
+cd $GEM_DQM_ROOT_BASE
+source bin/thisroot.sh
+cd -
+export LD_LIBRARY_PATH=/usr/local/gcc/4.9.3/lib64:$LD_LIBRARY_PATH
+export PATH=$PATH:$BUILD_HOME/gem-light-dqm/dqm-root/bin/linux/x86_64_slc6:$BUILD_HOME/gem-light-dqm/gemtreewriter/bin/linux/x86_64_slc6
```

Comment:
this has some `slc6` stuff leftover, is it because of a non-generic `Makefile`?

Reply:
Oups, actually this file can be safely removed at all. Not needed